tests/common/helpers/drop_counters/fanout_onyx_drop_counter.py

```diff
@@ -0,0 +1,75 @@
+import re
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+MAX_OPENFLOW_RULE_ID = 65535
+DEVICE_PORT_VLANS = 'device_port_vlans'
+TRUNK = 'Trunk'
+MODE = "mode"
+FAILED = 'failed'
+INVOCATION = 'invocation'
+STDOUT = 'stdout'
+
+
+class FanoutOnyxDropCounter:
+    def __init__(self, onyx_switch):
+        self.onyx_switch = onyx_switch
+        self.fanout_graph_facts = None
+
+    def prepare_config(self, fanout_graph_facts, match_mac, set_mac, eth_field):
```

Comment:
Suggest defining a base class to define `prepare_config` and `restore_config` APIs and have those specific implementations in the derivatives.